packages/nimble-components/src/table-column/base/cell-view/index.ts

```diff
@@ -29,6 +29,14 @@ export abstract class TableCellView<
     @observable
     public recordId?: string;
 
+    /**
+     * Gets the child elements in this cell view that should be able to be reached via Tab/ Shift-Tab,
+     * if any.
+     */
+    public get tabbableChildren(): HTMLElement[] {
```

Comment:
If a table column type will contain focusable/tabbable elements, our guidance would be:
- That column's cell view overrides this property, and returns the focusable element(s) in the array
- The focusable elements should be tabindex=-1 by default

`KeyboardNavigationManager` uses this when handling Tab/Shift-Tab for a given row, and to reflect the correct focus state if one of these elements is focused (`TableFocusType.cellContent`).

Currently `KeyboardNavigationManager` updates tabindex of these elements directly (except for the `tabIndexOverride` cases described previously). For arbitrary cell content, this might not be sufficient, since the element that should be tabindex=0 could be an element descendant in a shadow root or something similar. So we may need to add an additional API/method to handle focusing/unfocusing these children.